orchagent/portsorch.cpp

```diff
@@ -2384,6 +2416,15 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 {
                     mtu = (uint32_t)stoul(fvValue(i));
                 }
+                /* Set port TPID */
+                if (fvField(i) == "tpid")
+                {
+                    tpid_string = fvValue(i);
+                    // Need to get rid of the leading 0x
+                    tpid_string.erase(0,2);
```

Comment:
Is it better to use str.find and then str.erase(0, pos)? Do we always expect a `0x`?

Reply:
The CLI enforces that the TPID values must be one of the following: "0x8100", "0x9100", "0x9200", or "0x88A8".
So yes it will always have the "0x" which we can get rid of.